### ISSUE TITLE
[MDCAlertController] - no way to set background color 

### ISSUE THREAD

**Issue by muzoman** [material-components/material-components-ios#4594]
I've tried doing this:

`let colorScheme = MDCSemanticColorScheme()'

`colorScheme.surfaceColor = .black` 

`let alertRet = MDCAlertController(title: title, message: message)`

But the alert stays white. There doesn't seem to be a colorScheme attribute for background?

---
#### Internal data
- Associated internal bug: [b/117178308](http://b/117178308)

**Comment by romoore** [comment]
Thanks, @muzoman . Unfortunately there is no way to set the `backgroundColor` of an `MDCAlertController`.  Thanks for filing this bug.

Your example code is not quite right&mdash;we could probably do better in our documentation.  To apply a color scheme to an Alert Controller, you will need to use MDCAlertColorThemer:

```swift
let colorScheme = MDCSemanticColorScheme()
colorScheme.surfaceColor = .black

let alert = MDCAlertController(title: title, message: message)
MDCAlertColorThemer.applySemanticColorScheme(MDCSemanticColorScheme, to: alertController)
```

There is an example of this in [DialogsAlertViewController.m](https://github.com/material-components/material-components-ios/blob/develop/components/Dialogs/examples/DialogsAlertViewController.m#L75).

**Comment by romoore** [comment]
Hi @muzoman . An `MDCSemanticColorScheme` is just a data structure - it organizes the colors into semantic groupings and then the ColorThemers (for various components) apply the colors in those color schemes.  In our color themers,  `surfaceColor` is used as the default color for surfaces (like the background of cards or buttons), while `backgroundColor` is used as the default color for backgrounds (like the area behind CollectionView cells).  You can find some [more documentation about the color scheme in MDC-iOS on the material.io site](https://material.io/develop/ios/components/theming/color/).

**Comment by romoore** [comment]
Ah, and to clarify from my earlier post as well, you have filed a solid bug against MDC-iOS Dialogs. As far as I can tell, there is currently no way to customize the background color of the alert controller views.

**Comment by muzoman** [comment]
Ah, so the colorScheme stuff currently can't change the MDCAlertViewController background color. OK, so assume that this is then a feature request?

**Comment by romoore** [comment]
No, this is definitely a bug. I don't see any reason why you shouldn't be able to set the background color via theming. Certainly for darker color schemes a stark white background on an alert would be undesirable.

**Comment by muzoman** [comment]
Yes , Alerts in my App stand out like sore thumbs at present in my "dark mode" !

**Comment by romoore** [comment]
Fixed by #6066. You can now set `backgroundColor` on MDCAlertController.
